packages/core/config.js

```diff
@@ -90,6 +90,11 @@ module.exports.schema = {
         (accum, method) => accum && typeof value[method] === 'function',
         true
       ))
+  },
+  filters: {
+    defaultValue: () => [ 'password' ],
+    message: 'should be an array of strings',
+    validate: value => isArray(value) && value.length === filter(value, s => typeof s === 'string').length
```

Comment:
it'd be nice to cover this validation with a couple of unit tests if that hasn't already been done elsewhere.